bin/heudiconv

```diff
@@ -657,26 +668,36 @@ def convert_dicoms(subjs, dicom_dir_template, outdir, heuristic_file, converter,
                 "atm we support processing only 1 subject/session at a time"
             sess, sid = session_subject_ids.pop()
 
+
+        if is_bids and not sid.isalnum(): # alphanumeric only
+            old_sid = sid
```

Comment:
the problem is some of the dataset IDs have underscores and dashes, which would have to be changed in the dicom directories as well. If we just raise an error, that process will have to be done manually